client/src/app/appointments/appointments.component.ts

```diff
@@ -40,7 +46,28 @@ export class AppointmentsComponent {
    */
   public onHearingAidsEvaluation() {
     console.log('Hearing Aids Evaluation');
+    this.clearData();
     Utilities.setSessionStorage('appt', 'Hearing Aids Evaluation');
     this.router.navigateByUrl('/ts');
   }
+
+  private clearData(): void {
```

Comment:
This is duplicate code of a `clearData` elsewhere and would be better combined to re-use the logic here (but as one function that merely clears the data, and another function that first asks for confirmation per #90)

Reply:
I created an Injectable that any component can use to clear all patient/appointment data. You can specify whether or not to show a popup to allow the user to proceed.  This is currently only false when you click logout on audiologist navigation.